src/dev/chromium_version.ts

```diff
@@ -103,10 +112,29 @@ async function getChromiumCommit(
     throw new Error(`Could not find a Chromium commit! Check ${url} in a browser.`);
   }
 
-  log.info(`Found Chromium commit ${commit} from revision ${revision}.`);
+  log.debug(`Found Chromium commit ${commit} from revision ${revision}.`);
   return commit;
 }
 
+/**
+ * Get the Chrome release from git commit
+ */
+async function getChromiumRelease(commit: ChromiumCommit, log: ToolingLog) {
+  const url =
+    `https://storage.googleapis.com/chromium-find-releases-static/` +
+    `${commit.substring(0, 3)}.html#${commit}`;
+
+  log.info(`Fetching ${url}`);
+  const response = await fetch(url);
+  const pageText = await response.text();
+  const commitIndex = pageText.indexOf(commit);
+  if (commitIndex > 0) {
+    const substr = pageText.substring(commitIndex, commitIndex + 100);
+    const releaseVersion = substr.replace(/^.*"([\d\.]+)".*$/, '$1');
```

Comment:
Can we throw an error here if the replacement isn't successful (or a strict check on the release version string)?  If there's no match we'll end up trying to download whatever string is returned.  Not familiar with how the service handles error or our fetch chromium scrips.